packages/lab-extension/src/index.ts

```diff
@@ -15,11 +15,13 @@ import { INotebookTracker, NotebookPanel } from '@jupyterlab/notebook';
 
 import { ITranslator } from '@jupyterlab/translation';
 
-import { Menu, MenuBar } from '@lumino/widgets';
+import { Menu, MenuBar, Widget } from '@lumino/widgets';
 
 import { INotebookShell } from '@jupyter-notebook/application';
 
-import { caretDownIcon } from '@jupyterlab/ui-components';
+import { caretDownIcon, CommandToolbarButton } from '@jupyterlab/ui-components';
+
+import { openInNewIcon } from '@jupyter-notebook/ui-components';
```

Comment:
Looks like it should be possible to reuse the `launchIcon` icon from `@jupyterlab/ui-components` directly?

Which points to: https://github.com/jupyterlab/jupyterlab/blob/main/packages/ui-components/style/icons/toolbar/launch.svg

Reply:
That way we don't need to add the icon to the notebook code base.

Reply:
Good catch, it fixed the rendering of the icon as suggested, thank you

Reply:
![Screenshot 2023-05-18 at 3 51 42 PM](https://github.com/jupyter/notebook/assets/26686070/b24647ab-d21b-43a0-a69d-f5944f884548)